apps/src/lib/levelbuilder/script-editor/ScriptEditor.jsx

```diff
@@ -763,16 +807,12 @@ class ScriptEditor extends React.Component {
           )}
         </CollapsibleEditorSection>
 
-        <div style={styles.saveButtonBackground}>
-          <button
-            className="btn btn-primary"
-            type="submit"
-            style={styles.saveButton}
-            onClick={this.presubmit}
-          >
-            Save Changes
-          </button>
-        </div>
+        <SaveBar
+          handleSave={this.handleSave}
+          error={this.state.error}
+          isSaving={this.state.isSaving}
+          lastSaved={this.state.lastSaved}
+        />
```

Comment:
💯  nice job prepping this entire piece in a separate PR! not just for your reviewer's sake :)